src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -183,8 +195,15 @@ def _send_data_packet(
         data_to_send: bytes = bytes(0),
         truncate: bool = False,
     ) -> None:
+        # TODO Tanner (4/7/21): convert timestamp to microseconds once real board makes the switch
+        timestamp = (
+            (self._baseline_time_usec + _perf_counter_us())
```

Comment:
wouldn't `_perf_counter_us` need to be relative to the some timepoint? maybe I'm misunderstanding, but here wouldn't perf_counter just essentially be an arbitrary number?

Reply:
Yes it does need to be relative to a start timepoint